### PR TITLE
VUE-607 KvTextInput

### PR DESCRIPTION
**Pull request by ryan-ludwig**

### CODE REVIEW

File: @kiva/kv-components/vue/stories/KvTextInput.stories.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+	data: () => ({
+		textInputModel: '1640 Riverside Drive',
+	}),
+	template: `
+		<div>
+			<label
+				for="without-v-model-id"
+				class="tw-text-h4 tw-block tw-pb-1"
+			>
+				Street Adress
```
Comment (mcstover) [inline @kiva/kv-components/vue/stories/KvTextInput.stories.js:65]: `Address`

**Comment by ryan-ludwig** [comment]
Just a reminder, if you want to check out the Storybook of this PR, you can get there from the details link
image

**Review by eddieferrer — APPROVED** [review]

**Review by mcstover — APPROVED** [review]
Looks great!

**Review by BoulderBrains — APPROVED** [review]

**Review by emuvente — APPROVED** [review]
👍

**Pull request merged**